packages/core/lib/commands/debug/meta.js

```diff
@@ -27,6 +27,10 @@ module.exports = {
       describe: "Force debugger to skip compilation (dangerous!)",
       type: "boolean",
       default: false
+    },
+    "url": {
```

Comment:
Since you added this to debugger, I'm assuming you intended it as a debugger-specific thing.  If it's intended as a general feature then of course it'll have to go with the other general options.